approvalcrest-junit-jupiter-kotlin/src/test/kotlin/com/github/karsaig/approvalcrest/kotlin/matcher/KotlinMatchersTest.kt

```diff
@@ -0,0 +1,25 @@
+package com.github.karsaig.approvalcrest.kotlin.matcher
+
+import com.github.karsaig.approvalcrest.jupiter.MatcherAssert.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInfo
+
+class KotlinMatchersTest {
```

Comment:
Please create antoher module for integration tests for kotlin module and move this test to there. Similarly to other integration modules. Without those some changes broke approvalcrest, but every unit test passed. Integration modules make sure it can actually used in a project and it'll work as it should.

At this level, trying to move to use Jimfs for tests involving the filesystem.